third_party/rust/adss/v0_2/README.chromium

```diff
@@ -4,6 +4,6 @@ Description: Adept Secret Sharing framework
 Version: 0.2.2
 Security Critical: yes
 Shipped: yes
-License: Mozilla Public License 2.0
```

Comment:
Sorry, I meant to add a suggestion and accidentally committed directly to the branch instead :(